README.md

```diff
@@ -21,6 +23,10 @@ See [Background](Background.md) for the background information and development t
 
      + `Location` as a dependency for `ScanLocation`
 
+## Table definitions
+<details>
```

Comment:
I would limit click-to-expand format to cases where a user may not need this information. When it encompasses a whole section, it doesn't do much to facilitate navigation. In general, I would adopt the following structure

```
- Item 1. General item description. <details>
<summary>Click to expand details</summary>
   - Detail 1
   - Detail 2
</details>
- Item 2 General item description. <details>
<summary>Click to expand details</summary>
   - Detail 1
   - Detail 2
</details>
```  

For the current doc, I think the items could be (a) schema with collapsed table definitions and (b) installation methods, such as 'Install with Suite2p'

Reply:
Thanks @CBroz1.  I agree.  Will update this pull request.